src/Command/DrupalUpdate/UpdateScript.php

```diff
@@ -0,0 +1,183 @@
+   *
+   */
+  public function __construct() {
+    $this->checkinfo_obj = new CheckInfo();
+    $this->updateScriptUtility = new UpdateScriptUtility();
+    $this->setDir(getcwd());
+  }
+
+  /**
+   * @param array $latest_security_updates
+   */
+  public function updateAvailableUpdates(array $latest_security_updates) {
+    if(count($latest_security_updates)>1){
+      // if get any available then list out in tabular format.
+      $this->updateScriptUtility->printReleaseDetail($latest_security_updates);
+
+      // update current repo with updated code based on download and folder replace method on same path.
+      $this->updateScriptUtility->updateCode($latest_security_updates);
+
+      // remove the unwanted tar or downloaded files.
+      $this->updateScriptUtility->unlinktarfiles($latest_security_updates);
+    }else{
+      // If no updates found.
+      echo "\n Branch Upto date \n";
+    }
+  }
+
+  /**
+   * Scan the directory and sub directory recorsive.
+   * Get the list of all .info files list with path.
+   * @param string $dir
+   */
+  public function getInfoFilesList() {
+    $dir = $this->dir;
+    $scaned_files = array_diff(scandir($dir), ['.', '..']);
+    foreach($scaned_files as $child_dir){
+      $temp_path = $dir . "/" . $child_dir;
+      if(is_dir($temp_path)){
+        $scanned_file_path = array_diff(scandir($temp_path), ['.', '..']);
+        $this->checkinfo_obj->getFilesInfo($scanned_file_path, $temp_path);
+      }elseif($this->checkinfo_obj->endsWith($child_dir, '.info')){
+        if(isset($this->checkinfo_obj->info_files[$child_dir])){
+          if(is_array($this->checkinfo_obj->info_files[$child_dir])){
+            $this->checkinfo_obj->info_files[$child_dir][] = $dir . "/" . $child_dir;
+          }else{
+            $this->checkinfo_obj->info_files[$child_dir][] = $this->checkinfo_obj->info_files[$child_dir];
+            $this->checkinfo_obj->info_files[$child_dir] = $dir . "/" . $child_dir;
+          }
+        }else{
+          $this->checkinfo_obj->info_files[$child_dir] = $dir . "/" . $child_dir;
+        }
+
+      }
+    }
+  }
+
+  /**
+   * Get the detail information of info files
+   */
+  public function getPackageDetailInfo() {
+    // Get the detail information of module, theme.
+    // ( package, current version, project core or contrib).
+    foreach ($this->checkinfo_obj->info_files as $key => $value){
+      if(is_array($value)){
+        foreach ($value as $k => $v){
+          $this->checkinfo_obj->fileGetInfo($v, $key);
+        }
+      }else{
+        $this->checkinfo_obj->fileGetInfo($value, $key);
+      }
+    }
+  }
+
+  /**
+   * @return array
+   */
+  public function getAvailableUpdatesinfo() {
+    return $this->checkinfo_obj->available_updates;
+  }
+
+  /**
+   * @return array
+   */
+  public function getInfoFiles() {
+    return $this->checkinfo_obj->info_files;
+  }
+
+  public function securityUpdateVersion() {
+    if (!defined('DRUPAL_ROOT')) {
```

Comment:
Why run this in so many places? Pass the value in as an argument. Need to stop relying on globally namespaced variables.